os/mm/mm_heap/mm_getheap.c

```diff
@@ -43,7 +43,7 @@
 typedef struct app_heap_s {
 	struct app_heap_s *flink;
 	struct app_heap_s *blink;
-	char app_name[BIN_NAME_MAX];
```

Comment:
There are many locations which use BIN_NAME_MAX for array size. But you modify only here. If this is necessary, you should check all of others. But is this really necessary?